packages/svelte/src/app.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
```

Comment:
Is this `app.html` necessary if we're only building a library?

Reply:
Since I'm using SvelteKit it automatically creates this app folder, I'd like to keep it for testing, but maybe before 1.0 I'll delete it.

Reply:
Sure!